tests/utils.js

```diff
@@ -639,6 +667,45 @@ module.exports = {
     };
   },
 
+  /**
+   * Watches a given logfile until at least one line matches one of the given regular expressions.
+   * Watch expires after 10 seconds.
+   * @param {String} logFilename - filename of file in local logs directory
+   * @param {[RegExp]} regex - matching expression(s) run against lines
+   * @returns {Object} that contains the promise to resolve when logs lines are matched and a cancel function
+   */
```

Comment:
I was hoping that having this would make tests that were relying on known when updates have happened by waiting for the popup to appear. It does. 
But it makes all other tests that were doing ok by virtue of not waiting at all and just snugging in additional operations (like writing docs or whatnot) between updating settings and starting actual testing. 
Net loss is ~2 minutes / build, so I'm going to change this tomorrow to be optional. 
I needed to add this because on my new "medic/login/token" page, there's no popup to depend on.

Reply:
I've changed this so just that one test that required it waits for logs (I tried some more options but none of them were ideal and I didn't want to destabilize all e2e tests.